### PR TITLE
Use Correct Posthoc Approval Input

### PR DESCRIPTION
**Pull request by sbowenwilliams**
Correctly grab the `posthoc_approval_label` from input.
image

### CODE REVIEW

File: src/input.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export function getInput(): Input {
     slackHook: core.getInput('slack_hook', {
       required: true,
     }),
-    posthocApprovalLabel: core.getInput('retroactive_approval_label'),
+    posthocApprovalLabel: core.getInput('posthoc_approval_label'),
```
Comment (ebenoist) [inline src/input.ts:29]: classic posthoc move

**Comment by ebenoist** [comment]
Is there a test that could have validated this?

**Comment by sbowenwilliams** [comment]
There's currently no test around the input, but I did add the label to the `env` example so that when `run-development` is run a user can see the input is correctly grabbed.

**Comment by ebenoist** [comment]
Bummer, now I wish this was generic: https://github.com/actions/toolkit/blob/master/packages/core/src/core.ts#L69

Maybe a PR is in order?

**Comment by sbowenwilliams** [comment]
Yeah, I like that more than having to specify the keys we're looking for.

**Pull request merged**